docs/management/cli-rack-management.md

```diff
@@ -46,6 +46,8 @@ The parameters available for your Rack depend on the underlying cloud provider.
 | **node_type**                     | **t3.small**    |
 | **region**                        | **us-east-1**   |
 | **high_availability** *           | **true**        |
+| **power_off_schedule**            |                 |
+| **power_on_schedule**             |                 |
```

Comment:
Should we keep the same name as V2? `ScheduleRackScaleDown` and `ScheduleRackScaleUp`?
https://docsv2.convox.com/reference/rack-parameters#schedulerackscaledown-schedulerackscaleup